lib/plugins/aws/lib/naming.js

```diff
@@ -220,6 +220,12 @@ module.exports = {
       .getNormalizedFunctionName(functionName)}EventsRuleCloudWatchEvent${cloudWatchIndex}`;
   },
 
+  // CloudWatch Log
+  getCloudWatchLogLogicalId(functionName, logsIndex) {
+    return `${this
+      .getNormalizedFunctionName(functionName)}CloudWatchLog${logsIndex}`;
```

Comment:
This should be renamed from `{functionName}CloudWatchLogs{logsIndex}` to `{functionName}LogsSubscriptionFilterCloudWatchLog{logsIndex}`

since we're using the `AWS::Logs::SubscriptionFilter` resource.

This way it's more predictable for the user.

The `cloudwatchEvent` event e.g. uses a `AWS::Events::Rule` resource and therefore the name of the generated resource is `{normalizedFunctionName}EventsRuleCloudWatchEvent{sequentialId}`

See: https://serverless.com/framework/docs/providers/aws/guide/resources/